playbooks/roles/swapfile/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Do you mind putting blank lines between each task? I find it helps with readability, and yes its fair to ask for a lint rule around this.